src/Components/Notification.vue

```diff
@@ -141,73 +162,74 @@ export default {
 			return (new Date(this.datetime)).valueOf()
 		},
 		absoluteDate: function() {
-			return OC.Util.formatDate(this.timestamp)
+			return OC.Util.formatDate(this.timestamp, 'LLL')
```

Comment:
`OC.Util.formatDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment`